src/runtime/posix_allocator.cpp

```diff
@@ -10,13 +10,13 @@ extern void free(void *);
 namespace Halide { namespace Runtime { namespace Internal {
 
 WEAK void *default_malloc(void *user_context, size_t x) {
-    void *orig = malloc(x+40);
+    void *orig = malloc(x+136);
```

Comment:
We really should change this to use a constant instead of hardcoded literals, e.g.:

    const size_t alignment = 128;
    ...
    void *orig = malloc(x + alignment + sizeof(void *));
    ...
    void *ptr = (void *)((((size_t)orig + alignment) & ~(alignment - 1));